.github/workflows/publish_docs.yml

```diff
@@ -1,57 +1,37 @@
 name: Publish docs to pages
 run-name: ${{ github.actor }} is publishing docs
 on:
-    # on a successful push
+    workflow_dispatch:  # Allow manual triggering
     push:
-        # only run on pushes to these branches
-        branches:
-            - 'master'
-            - 'main'
-            - 'docs'
-        # or run if pushes to these files.
-        # paths:
-        #    - 'docs/**.md'
-    check_run:
-        types:
-            # Only run if we have a successfully completed branch
-            - completed
```

Comment:
Hrm, interesting. I'm wondering if this is also related to the permissions that we have configured, so that external folk don't trigger certain builds. That may explain why that step doesn't fail for us.